### ISSUE TITLE
1.15.x refers to obsolete OBJLoader.addDomain()

### ISSUE THREAD

**Issue by LemADEC** [MinecraftForge/Documentation#369]
As seen here: https://mcforge.readthedocs.io/en/1.15.x/models/files/#wavefront-obj-models , we're supposed to use OBJLoader.addDomain to register our namespace where WaveFront object support will be added.
This doesn't apply since 1.15.x, instead we should add a loader entry to the model's json file, like so:
```json
{
	"parent": "block/cube_all",
	"loader": "forge:obj",
	"model": "yournamespace:models/block/xxx/yyy.obj"
}
```

**Comment by ChampionAsh5357** [comment]
Already referenced by #283. The entire files page needs to be rewritten to support the current model loader system. You're welcome to rewrite and supply this to the omnibus.

**Comment by ChampionAsh5357** [comment]
Merged into omnibus #364.
